docs/external.rst

```diff
@@ -1,68 +1,140 @@
 .. _external:
 
-Interacting with external arrays
-================================
+Exporting and importing data
+============================
```

Comment:
I actually think the old title is clearer. "Data" can refer to many things, and people may mess up this part with PLY/mp4 exporting if we use the new title.

Reply:
```suggestion
Interacting with external arrays
================================
```